src/Authentication/Controllers/RequestSystemUserController.cs

```diff
@@ -95,6 +95,7 @@ public async Task<ActionResult<RequestSystemResponse>> CreateRequest([FromBody]
         Result<RequestSystemResponse> response = await _requestSystemUser.GetRequestByExternalRef(externalRequestId, vendorOrgNo);
         if (response.IsSuccess)
         {
+            response.Value.ConfirmUrl = CONFIRMURL1 + _generalSettings.HostName + CONFIRMURL2 + response.Value.Id;
```

Comment:
Do we have tests for this?

Reply:
The ConfirmUrl is for the auth.ui, and is not stored in the repository.
We had a test for the first request, not for a subsequent request with the same variables. 
Will add one.